api/app/tests/Behat/features-v2/registration/ingest.lay.sirius.feature

```diff
@@ -1,42 +1,44 @@
-@v2 @v2_sequential_1 @registration @ingest
```

Comment:
I would strongly suggest not commenting out these behat tests as they are testing a lot of our business logic around registering and managing cases.

We should refactor the frontend forms or these tests so they align with our CSV upload process before merging in these changes if these tests are failing. 

I do agree with you that we shouldn't have forms or fields just for tests if we can avoid it.

Reply:
Will pick up the ticket I've created in the new year then. Off the back of this, as it's the first time I've really looked at our Behat tests. Is there a reason as to why the folder structure for these in buried into the api folder & not in it's own top level tests/behat folder? There was a moment of confusion when looking at this, with it touching the FE & API but is buried inside the API folder?

We shouldn't have them full stop. If we need to consider this then either our approach is wrong or how we are attempting to test is wrong. This should not have gone out once the process was refactored.